CHANGELOG.md

```diff
@@ -221,6 +221,316 @@
   * Reset upstream branch to dev version
   * If destination does not exists, do not throw error
 
+## v1.22.0 (2021-08-02)
```

Comment:
Why does this duplicate some of the content that's above it, implying that this release is older than the releases described above?

Reply:
IDK @nalind.  The changelog.txt was funky.  After I ran the release.sh file, it showed only the entries for 1.19 and lower, so I added all the rest back in, but you're right, its duplicated here now.

Reply:
And actually, this is CHANGELOG.md, not changelog.txt.  I must have simply cut/pasted badly.  If this is the only change needed, would you be OK with merging this in now and then I'll follow up and change this file.  I don't think it's included in the rpm, only shown on GitHub.